app/views/payments/new.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= render partial: "shared/navigation", locals: {icon_type:'fa-solid', icon_name: 'arrow-left', title: 'NEW TRANSACTION', right:''} %>
+<%= render partial: "form", locals: {payment: @payment, group: @group} %>
```

Comment:
- [ ]  On the add a new transaction/payment page, please replace your Home button with a "Back" button (<) as stipulated in the project requirements so that when the user clicks on the "Back" button (<), the user navigates to the transactions page for that category

Reply:
👍